CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Unreleased
+## 1.154.0 (December, 12 2023)
+NOTES:
+* documentation: resource/spotinst_elastigroup_aws: Added 2 params (`period` and `evaluation_periods`) as part of target scaling policy.
```

Comment:
please remove parenthesis and
mention `scaling_target_policy` instead of target scaling policy

Reply:
resolved